sql/parser/sql.y

```diff
@@ -1402,11 +1402,11 @@ rename_stmt:
   }
 | ALTER INDEX qualified_name RENAME TO name
   {
-    $$ = nil
+    $$ = &RenameIndex{Name: $3, NewName: Name($6), IfExists: false}
   }
```

Comment:
@MBerhault  Could you write the complete grammer aias, I did some search, didn't find it.